### PR TITLE
UIDATIMP-55: Implement reset file extensions feature

### PR DESCRIPTION
**Pull request by VictorSoroka1**

### CODE REVIEW

File: test/bigtest/network/scenarios/fetch-file-extensions-success.js [hunk 1]
```diff
@@ -1,5 +1,11 @@
-export default server => {
-  server.createList('file-extension', 3);
+export default async server => {
```
Comment (OleksandrYehorovEPAM) [inline test/bigtest/network/scenarios/fetch-file-extensions-success.js]: Does it need to be async? (here and in other scenarios)

Comment (VictorSoroka1) [reply test/bigtest/network/scenarios/fetch-file-extensions-success.js]: Can get rid of that.

File: src/components/JobLogs/withJobLogsSort.js [hunk 2]
```diff
@@ -95,8 +95,9 @@ const withJobLogsSortComponent = WrappedComponent => {
           useFormatterFn,
           sortFn,
         } = this.sortColumns[sort];
+        const useCellFormatter = useFormatterFn && cellFormatter;
 
-        return sortDir * (useFormatterFn && cellFormatter ?
+        return sortDir * (useCellFormatter ?
```
Comment (OleksandrYehorovEPAM) [inline src/components/JobLogs/withJobLogsSort.js]: Let's write ternary operators like described in rules (point 9): https://wiki.folio.org/display/FOLIJET/Code+style+rules+not+covered+by+eslint

File: src/settings/FileExtensions/FileExtensions.js [hunk 3]
```diff
@@ -144,17 +168,76 @@ class FileExtensions extends Component {
...
+      this.setState({ areFileExtensionsResetting: true });
+      await restoreDefaultFileExtensions.POST({});
+
+      this.transitionToParams({
+        _path: `${path}/view`,
+        layer: null,
+      });
+
+      this.hideResetFileExtensionsToDefaultsModal();
+    } catch (e) {
```
Comment (OleksandrYehorovEPAM) [inline src/settings/FileExtensions/FileExtensions.js]:
```suggestion
    } catch (error) {
```

**Pull request merged**